lab 2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>My title</title>
+
+      <!-- font awesome -->
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+   </head>
+   <body>
+   <header>
+      <div class="collapsed-menu">
+         <div class="title menu-item">Navigation</div>
+         <div class="button menu-item"><i class="fa fa-bars" aria-hidden="true"></i></div>
+      </div>
+      <div class="menu">
+         <div class="menu-item"><a href='#'>psd websites</a></div>
+         <div class="menu-item"><a href='#'>ui kits</a></div>
+         <div class="menu-item"><a href='#'>premium psd files</a></div>
+         <div class="menu-item"><a href='#'>about</a></div>
+         <div class="menu-item"><a href='#'>blog</a></div>
+      </div>
+   </header>
+   <article>
+      <div class="page page-1">
+         <section>
+            <div class="content-main">
+               <h1>Awesome Free PSD <br> templates &amp; Resources</h1>   
+            </div>
+            <a href="#">
+               <div class="link" style="display: flex; justify-content: space-between; align-items: center;">
```

Comment:
Could be just:
span > www.BlazRobar.com < i >...< / i > < / span >

Reply:
it seems to me, that it's better to use flex with conteiner-like tags (e.g. div, container, nav), isn't it?

Reply:
Okay then, what's the reason to use inline styles?

Reply:
it seemed to be shorter. 
In my oppinion, flex-things are more connected to document flow but styles (especially in that case, were that position of link with arrow relatively to each other doesn't seem to be able to change with time).

Reply:
Looks like an interesting discussion, I'm kindly asking you to attend our next classes to have a talk about it

Reply:
Well, i have no other choices :)